install.sh

```diff
@@ -169,6 +170,13 @@ setup_homebrew() {
     "$(brew --prefix)"/opt/fzf/install --key-bindings --completion --no-update-rc --no-bash --no-fish
 }
 
+function setup_nvim() {
```

Comment:
**praise:** I like the idea of adding a `setup_nvim` section to the installer, but this seems out of place based on the description of this PR.